public/apps/configuration/app-router.tsx

```diff
@@ -38,31 +38,37 @@ import { TenantList } from './panels/tenant-list/tenant-list';
 import { UserList } from './panels/user-list';
 import { Action, RouteItem, SubAction } from './types';
 import { ResourceType } from '../../../common';
-import { buildHashUrl, buildUrl } from './utils/url-builder';
+import { buildUrl } from './utils/url-builder';
 import { CrossPageToast } from './cross-page-toast';
 import { getDataSourceFromUrl, LocalCluster } from '../../utils/datasource-utils';
+import { getBreadcrumbs } from './utils/resource-utils';
 
 const LANDING_PAGE_URL = '/getstarted';
 
 export const ROUTE_MAP: { [key: string]: RouteItem } = {
   getStarted: {
     name: 'Get Started',
+    displayNameWithoutSecurityBase: 'Get started with access control',
```

Comment:
How come we are displaying security specific naming without the Security plugin? Or am I misinterpreting `displayNameWithoutSecurityBase`

Reply:
Updated the name of the variable to be more explicit. This is for what breadcrumb to show when the plugin is split into different apps for each tab (new navigation). I changed the variable name to breadCrumbDisplayNameWithoutSecuirtyBase to make this clear.